pkg/plugin/name.go

```diff
@@ -72,18 +72,29 @@ func (fn FullName) PluginVersion() string {
 	return PluginVersionLatest // default
 }
 
-func (fn FullName) PluginVersionGreaterThan(other FullName) bool {
-	leftVersion := fn.PluginVersion()
-	rightVersion := other.PluginVersion()
+func (fn FullName) PluginVersionGreaterThan(right FullName) bool {
+	if right == "" {
+		return true // right is empty, left is greater either way
+	}
+	if fn == "" {
+		return true // left is empty, right is greater either way
+	}
```

Comment:
I understand this could be called either from a processor or a connector, maybe we could simplify it this way, what do you think?

```suggestion
	if right == "" || fn == "" {
                return true // if either right or left is empty, the other is greater either way
         }
```

Reply:
yeah, we could do that, if that wouldn't be a typo :D the second branch should return `false` 🙈